uber/templates/registration/register.html

```diff
@@ -1,131 +1,64 @@
-<!DOCTYPE HTML>
```

Comment:
> What does this actually look like? Wouldn't it include things we don't want, like kick-in levels? That's more an issue for Prime than for Magstock.

In this case I manually removed the kick-in rows using some jQuery (I'll leave a comment below showing where I did that).  I believe that all of the fields being shown are ones we want to have, though perhaps we want to omit volunteer signup in at-the-door changes.

Come to think of it, I was assuming we would want volunteer signup in at-the-door reg, but come to think of it we just tell people to come to Stops, so I should probably remove that as well.  I'll handle that in a separate commit in this branch in a few minutes.


> Does this mean in theory that we could use at-door reg stations for Magstock?

Yep, that's the plan.  Is that not what you did last year?  I was assuming you'd want that this year.  Either way, it'll be working in case you want/need it, and you can ignore it if you don't care/want it this time around.